src/main/webapp/app/pages/newsPage/NewsPageContent.tsx

```diff
@@ -289,6 +289,51 @@ const EVIDENCE_COLUMN_SEPARATOR = '; ';
 // https://stackoverflow.com/questions/41947168/is-it-possible-to-use-keyof-operator-on-literals-instead-of-interfaces
 
 export const NEWS_BY_DATE: { [date: string]: NewsData } = {
+  '09192024': {
+    changedAnnotations: [
+      {
+        columnHeaderType: AnnotationColumnHeaderType.NEW_ALTERATION_WITH_LEVEL,
+        content: [
+          [
+            '3A',
+            'ALK',
+            'F1174L/C, F1245Y/V, R1275Q/L',
```

Comment:
Could you verify if this will be converted to alteration page links? If not, we might need to use `WithSeparator`.